### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by thesergiomiguel**

### CODE REVIEW

File: src/components/RenderAs/RenderAs.tsx [hunk 1]
```diff
@@ -13,10 +13,9 @@ class RenderAs extends React.PureComponent<Props> {
       children,
       ...props
     } = this.props;
-    const TagName = as;
+    const TagName = as as React.ElementType<Props>;
```
Comment (tomtt) [inline src/components/RenderAs/RenderAs.tsx]: as as?

Comment (thesergiomiguel) [reply src/components/RenderAs/RenderAs.tsx]: :D

Comment (thesergiomiguel) [reply src/components/RenderAs/RenderAs.tsx]: But yes, and it works 🧙‍♀️

Comment (tomtt) [reply src/components/RenderAs/RenderAs.tsx]: :shipit:

**Pull request merged**